### DISCUSSION TITLE
What format for the SD Card?   "Checking SDCARD...Error!"

### DISCUSSION THREAD

**Discussion by LeifBloomquist** [zeldin/OrangeCart#7] [Super-REU]
Playing around with the super-reu 1.2 bitstream, got the m64 files from the Initial release unzipped and copied to an SD Card just fine.

Unfortunately I'm getting **Checking SDCARD...Error!   Please remove SDCARD**  from the movie player app.

The card is formatted as FAT32, should it be something else?

**Comment by zeldin** [comment]
FAT32 is fine, long filenames (VFAT) are also supported.
However, the error message you are seeing is not related to the filesystem, it shows that it was the initialization of the card itself that failed.  The initial handshaking with an SDcard is a bit fiddly, and different cards seem to have different quirks.  Things that you can try:
* Remove and reinsert the card a couple of times
* Have the card inserted already before entering the player
* If possible, try a different brand of card

If there is some specific kind of card causing troubles it may be possible to make a workaround for it.
